Sources/AccountSDKIOSWeb/Client/Client.swift

```diff
@@ -230,6 +238,30 @@ public class Client: CustomStringConvertible {
         }
     }
     
+    // Used only for Simplified Login feature with additional parameter window
```

Comment:
I am not happy about this part of code (duplicated). I needed to pass UiWindow to the Client to have the right window to dismiss the Simplified Login UI... On the other hand, i don't want to make window parameter public in methods getLoginSession

Reply:
Yeaa this was a thinker🤔

Reply:
I do not see any other solution for now without unnecessary changes in public api